doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -1154,9 +1154,19 @@ Info
 Show Dataverse Version and Build Number
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-|CORS| Get the Dataverse version. The response contains the version and build numbers::
+|CORS| Get the Dataverse version. The response contains the version and build numbers:
 
-  GET http://$SERVER/api/info/version
```

Comment:
Here it would be nice to add this:

.. note:: See :ref:`curl-examples-and-environment-variables` if you are unfamiliar with the use of ``export`` below.